lib/core/viewer/components/viewer.component.html

```diff
@@ -101,61 +101,56 @@ <h2>{{ 'ADF_VIEWER.LOADING' | translate }}</h2>
             </div>
         </ng-container>
 
-        <div *ngIf="!isLoading" class="adf-viewer-layout">
-
-            <ng-container *ngIf="showSidebar && sidebarPosition === 'left'">
-                <div class="adf-viewer__sidebar adf-viewer__sidebar-left">
-                    <ng-content select="adf-viewer-sidebar"></ng-content>
-                    <ng-container *ngIf="!sidebar">
-                        <!-- todo: default info drawer -->
-                    </ng-container>
-                </div>
-            </ng-container>
-
-            <div class="adf-viewer-layout-content">
-                <div class="adf-viewer-content-container" [ngSwitch]="viewerType">
-
-                    <ng-container *ngSwitchCase="'pdf'">
-                        <adf-pdf-viewer [blobFile]="blobFile" [urlFile]="urlFileContent" [nameFile]="displayName"></adf-pdf-viewer>
-                    </ng-container>
-
-                    <ng-container *ngSwitchCase="'image'">
-                        <adf-img-viewer [urlFile]="urlFileContent" [nameFile]="displayName" [blobFile]="blobFile"></adf-img-viewer>
-                    </ng-container>
-
-                    <ng-container *ngSwitchCase="'media'">
-                        <adf-media-player [urlFile]="urlFileContent" [mimeType]="mimeType" [blobFile]="blobFile" [nameFile]="displayName"></adf-media-player>
-                    </ng-container>
-
-                    <ng-container *ngSwitchCase="'text'">
-                        <adf-txt-viewer [urlFile]="urlFileContent" [blobFile]="blobFile"></adf-txt-viewer>
-                    </ng-container>
-
-                    <ng-container *ngSwitchCase="'custom'">
-                        <span *ngFor="let extensionTemplate of extensionTemplates">
-                            <ng-template
-                                *ngIf="extensionTemplate.isVisible"
-                                [ngTemplateOutlet]="extensionTemplate.template"
-                                [ngTemplateOutletContext]="{ urlFileContent: urlFileContent, extension:extension }">
-                            </ng-template>
-                        </span>
-                    </ng-container>
-
-                    <ng-container *ngSwitchDefault>
-                        <adf-viewer-unknown-format></adf-viewer-unknown-format>
-                    </ng-container>
-
-                </div>
+        <div *ngIf="!isLoading"  fxLayout="row" >
+            
+            <div *ngIf="showSidebar" fxFlexOrder="{{sidebarPosition === 'left'? 1: 2 }}" >
+                <ng-container>
+                    <div class="adf-viewer__sidebar">
+                        <ng-content select="adf-viewer-sidebar"></ng-content>
+                        <ng-container *ngIf="!sidebar">
+                            <!-- todo: default info drawer -->
+                        </ng-container>
+                    </div>
+                </ng-container>
             </div>
-
```

Comment:
Please have a more proper look on this rebase, it is not correct. In the meantime the original functionality changed, and with this rebase that new functionality is gone.